### ISSUE TITLE
[FEATURE] Find a way to deal with styling/controlling child components

### ISSUE THREAD

**Issue by filrak** [vuestorefront/storefront-ui#1363]
```js
  <SfBanner
    :button="{ size: 'xl', styles: 'c-pink, font-bold' }"
  >
    <SfButton>
  <SfBanner>
```

┆Issue is synchronized with this [Jira Zadanie](https://vsf.atlassian.net/browse/SFUI-31) by [Unito](https://www.unito.io)

**Comment by justyna-13** [comment]
Already in progress #1399

**Comment by mayashavin** [comment]
Locked since this issue is going to be fix in #1399
